apps/fz_http/test/support/fixtures/configurations_fixtures.ex

```diff
@@ -188,4 +188,43 @@ defmodule FzHttp.ConfigurationsFixtures do
       %{"id" => "test", "label" => "SAML"}
     ]
   end
+
+  def saml_metadata do
```

Comment:
We already have it in `FzHttp.SAMLIdentityProviderFixtures`. Maybe format that one instead and expose metadata as a function?

Reply:
Ah, ok.

Reply:
Fixed